jenkins/catapult/runDockerBuildInnerBuild.py

```diff
@@ -158,7 +159,7 @@ def main():
 
 	builder = BuildManager(args, process_manager, environment_manager)
 	env = LinuxEnvironment(builder.use_conan, process_manager, environment_manager)
-	env.prepare()
+	env.prepare(builder.enable_code_coverage, args.source_path)
```

Comment:
why is dest dir `source_path/_build`? 😬 
(isn't it possible to make build inside /tmp, like earlier?)